src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    public static final int FIGURES_AMOUNT = 7;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[FIGURES_AMOUNT];
+        figures[0] = figureSupplier.getRectangle();
```

Comment:
This solution is hardcoding, think about how you can implement task description in other way: The first half of figures in this array should be generated with random parameters.